code/game/objects/items/crab17.dm

```diff
@@ -30,6 +30,13 @@
 			var/datum/bank_account/B = i
 			B.being_dumped = TRUE
 		new /obj/effect/dumpeet_target(targetturf, L)
+
+		to_chat(user, span_notice("You have activated Protocol CRAB-17."))
+		message_admins("[ADMIN_LOOKUPFLW(user)] has activated Protocol CRAB-17.")
+		log_admin("[key_name(user)] has activated Protocol CRAB-17.")
+		user.log_message("activated Protocol CRAB-17.", LOG_GAME)
+		deadchat_broadcast("[user] has activated Protocol CRAB-17.", message_type=DEADCHAT_ANNOUNCEMENT)
```

Comment:
Also a little unsure that this line is necessary, given that we have https://github.com/tgstation/tgstation/blob/b4fac6a903a6d5a1c43b679e4510f91535c19202/code/game/objects/items/crab17.dm#L240

Reply:
Removed with https://github.com/tgstation/tgstation/pull/69522/commits/ad5cba1b6ec92a458edd5ae7535cd11e8a943638